src/NzbDrone.Core/Localization/Core/en.json

```
@@ -929,6 +929,7 @@
   "RemotePathMappingRemotePathHelpText": "Root path to the directory that the Download Client accesses",
   "RemotePathMappingWrongOSPathHealthCheckMessage": "Remote download client {0} places downloads in {1} but this is not a valid {2} path. Review your remote path mappings and download client settings.",
   "RemotePathMappings": "Remote Path Mappings",
+  "RemotePathMappingsInfo": "Remote Path Mappings are very rarely required, it is better to match your download path client to your local paths.",
```

```suggestion
  "RemotePathMappingsInfo": "Remote Path Mappings are very rarely required, if {app} and your download client are on the same system it is better to match your download client path. For more information see {wikiLink}",
```

I don't think we need to spell out every possible combination in the info text, if people don't want to follow a link to the wiki they're probably not reading it in the UI either.

---

incorporated this with a slight tweak

---

Troubleshooting section has more details and links back to settings.

---

> Troubleshooting section has more details and links back to settings.

not sure we should be directly sending someone to troubleshoot for general info about what Remote Path Mappings are